dense_retriever.py

```diff
@@ -235,11 +216,11 @@ def main(args):
     input_paths = glob.glob(ctx_files_pattern)
 
     index_path = "_".join(input_paths[0].split("_")[:-1])
-    if args.save_or_load_index and os.path.exists(index_path):
+    if args.save_or_load_index and (os.path.exists(index_path) or os.path.exists(index_path + ".index.dpr")):
```

Comment:
I forgot to separate the commit; this line is to load the index file saved with this path: https://github.com/facebookresearch/DPR/blob/a6296df83644ae00c1aa62884c0cdb2be347a547/dpr/indexer/faiss_indexers.py#L56
which is the case that there exists no directory with the name of the value of `file` variable.